src/drivers/amqp_interop/Queue.php

```diff
@@ -221,12 +221,6 @@ class Queue extends CliQueue
      * @var string
      */
     public $driver = self::ENQUEUE_AMQP_LIB;
-    /**
-     * This property should be an integer indicating the maximum priority the queue should support. Default is 10.
-     *
-     * @var int
-     */
-    public $maxPriority = 10;
```

Comment:
Can't be merged w/o breaking backwards compatibility with current way the component is configured.

Reply:
I tried to keep backward compatibility
https://github.com/yiisoft/yii2-queue/pull/463

Reply:
@s1lver what do you think?

Reply:
We could add a description to UPGRADE.md
in my opinion this is a normal solution, only in this case it is not necessary to add options `'x-max-priority' => 10` to `$queueOptionalArguments`. This array must be empty

Reply:
Ready